articles/custom-domains/additional-configuration.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Additional Configuration for Custom Domains
+description: Describes the configuration steps you might need to follow in order to set up custom domains, depending on the Auth0 features you are using
+toc: true
+---
+# Additional Configuration for Custom Domains
+
+In order to set up custom domains, and depending on the Auth0 features you are using, there might be additional configuration steps you must follow. This article lists the required configuration you must do per feature.
+
+If you have been using Auth0 for some time and decide to enable a custom domain, you will have to migrate your existing apps and update the settings as described below. Note that existing sessions created at `${account.namespace}` will no longer be valid once you start using your custom domain, so users will have to login again.
+
```

Comment:
Do you think it's a good idea to have a "pre-requisite" section in here, e.g. "Pre-req: you have configured and verififed your custom domain, per the documentation" with a link to the documentation https://auth0.com/docs/custom-domains#how-to-configure-custom-domains? That way if customers find their way to this doc, they can easily figure out where they should start before doing these feature-specific setups. Wdyt?

Reply:
Good idea, I'll add this to its own section since lots of people skip intros.